### PR TITLE
Fixed Disolving FavTriggersFragment into TriggersFragment #147

### PR DESCRIPTION
**Pull request by pg301**

### CODE REVIEW

**Comment by pg301** [comment]
@garg-anuj @danielpassos @pilhuhn Please review this.

File: mobile/src/main/java/org/hawkular/client/android/fragment/TriggersFragment.java [hunk 1]
```diff
@@ -114,14 +116,19 @@ public void onRefresh() {
     }
 
     private void setUpTriggersForced() {
-        BackendClient.of(this).getTriggers(new TriggersCallback());
+
+        if( getArguments().getString("state").compareTo("From Favourite") == 0) {
```
Comment (KeenWarrior) [inline mobile/src/main/java/org/hawkular/client/android/fragment/TriggersFragment.java]: Why not use .equals?

Comment (pg301) [reply mobile/src/main/java/org/hawkular/client/android/fragment/TriggersFragment.java]: Yeah! we can do that

Comment (pg301) [reply mobile/src/main/java/org/hawkular/client/android/fragment/TriggersFragment.java]: Some more changes @garg-anuj

Comment (KeenWarrior) [reply mobile/src/main/java/org/hawkular/client/android/fragment/TriggersFragment.java]: Nothing else for now if it is working fine

**Review by KeenWarrior — APPROVED** [review]
LGTM :+1:

**Pull request merged**